README.md

```diff
@@ -8,6 +8,7 @@ Lista em ordem alfabética.
 | [![Bio Ritmo](https://github.com/lucascaton/empresas-brasileiras-usando-ruby/blob/master/companies/bio-ritmo.png)](https://www.bioritmo.com.br)                   | [bioritmo.com.br/trabalhe_conosco](https://www.bioritmo.com.br/trabalhe_conosco) |       | São Paulo-SP                                                                                                                                                             |         |
 | [![CloudWalk](https://github.com/lucascaton/empresas-brasileiras-usando-ruby/blob/master/companies/cloud-walk.png)](https://www.cloudwalk.io/pt)                  | [cloudwalk.io/pt/careers](https://www.cloudwalk.io/pt/careers)                   |       | São Paulo-SP                                                                                                                                                             |         |
 | [![Codeminer 42](https://github.com/lucascaton/empresas-brasileiras-usando-ruby/blob/master/companies/codeminer.png)](https://www.codeminer42.com)                | [become@codeminer42.com](mailto:become@codeminer42.com)                          | 65    | São Paulo-SP<br>Campinas-SP<br>Novo Hamburgo-RS<br>Natal-RN<br>Sorocaba-SP<br>Teresina-PI<br>Anápolis-GO<br>Goiânia-GO<br>Batatais-SP<br>Guarapuava-PR<br>Santa Maria-RS |         |
+| [![Defensoria Pública do Tocantins](/companies/dpeto.png?raw=true)](http://defensoria.to.def.br)| | 15 | Palmas-TO | ❌
```

Comment:
Today I learned that you can use `raw=true`, nice!
I'll update the other items as well :)